docs/_docs/quickstart.md

```diff
@@ -4,25 +4,37 @@ title: Quick-start guide
 permalink: /docs/quickstart/
 ---
 
-For the impatient, here's how to get a boilerplate Jekyll site up and running.
+If you already have [Ruby](https://www.ruby-lang.org/en/downloads/) and [RubyGems](https://rubygems.org/pages/download) installed (see Jekyll's [requirements](/docs/installation/#requirements/)), you can create a new Jekyll site by doing the following:
 
 ```sh
+# Install Jekyll and Bundler gems through RubyGems
 ~ $ gem install jekyll bundler
+
+# Create a new Jekyll site at ./myblog
 ~ $ jekyll new myblog
+
+# Change into your new directory
 ~ $ cd myblog
+
+# Build the site on the preview server
 ~/myblog $ bundle exec jekyll serve
-# => Now browse to http://localhost:4000
+
+# Now browse to http://localhost:4000
 ```
 
-The `jekyll new` command now automatically initiates `bundle install` and installs the dependencies required. To skip this, pass `--skip-bundle` option like so `jekyll new myblog --skip-bundle`.
+`gem install jekyll bundler` installs the [jekyll](https://rubygems.org/gems/jekyll/) and [bundler](https://rubygems.org/gems/bundler) gems through [RubyGems](https://rubygems.org/). You need only to install the gems one time &mdash; not every time you create a new Jekyll project. Here are some additional details:
+
+* `bundler` is a gem that manages other Ruby gems. It makes sure your gems and gem versions are compatible, and that you have all necessary dependencies each gem requires.
```

Comment:
This page aims to give a few instructions to get going. Detailed explanations are fine, we could split this in two sections:

```markdown
## For the impatient
...
## For the curious
```